.github/workflows/scripts/pre-publish-hook

```diff
@@ -4,59 +4,16 @@ set -Eeuo pipefail
 # remove "Debug|Any CPU" from the list so that "Release|Any CPU" is the default
 f=Shippo.sln
 echo updating $f...
-sed -i "/^[[:blank:]]*Debug|Any CPU.*/d" $f
+sed -i "/^[[:blank:]]*Debug|Any CPU.*/d" "$f"
 
 # add various metadata
 # modify/remove a few generated parts
 f=Shippo/Shippo.csproj
 echo updating $f...
-condition="'\$(GITHUB_ACTIONS)' == 'true'"
 yq -i -o xml -p xml '
   ( del( .Project.ItemGroup[0] | select(.+@Include == "..\docs\**\*") )
-  | .Project.PropertyGroup = [
-      .Project.PropertyGroup,
-      [
-        {
-          "+@Condition": "'"$condition"'",
-          "ContinuousIntegrationBuild": "true",
-          "EmbedUntrackedSources": "true",
-          "IncludeSymbols": "true",
-          "PublishRepositoryUrl": "true",
-          "SymbolPackageFormat": "snupkg"
-        },
-        {
-          "Copyright": "Copyright (c) Shippo 2024",
-          "Description": ".NET library which integrates with Shippo Multi Carrier Shipping API. This library provides access to Shippo (goshippo.com) API capabilities. Including label generation, rating, tracking and more.",
-          "PackageLicenseFile": "LICENSE",
-          "PackageIcon": "shippo.png",
-          "PackageProjectUrl": "https://github.com/goshippo/shippo-csharp-sdk",
-          "PackageTags": "USPS Fedex UPS API Shippo",
-          "RepositoryType": "git"
-        }
-      ]
-    ]
-  | .Project.ItemGroup = [
-      .Project.ItemGroup,
-      {
-        "None": [
-          {
-            "+@Include": "..\LICENSE",
-            "+@Pack": "true",
-            "+@PackagePath": "\\"
-          },
-          {
-            "+@Include": "..\res\shippo.png",
-            "+@Pack": "true",
-            "+@PackagePath": "\\"
-          }
-        ],
-        "PackageReference": {
-          "+@Include": "Microsoft.SourceLink.GitHub",
-          "+@PrivateAssets": "All",
-          "+@Version": "8.0.0"
-        }
-      }
-    ]
-  )' $f
+  )' $f &&
+sed -i "s/<EmbedAllSources>true<\/EmbedAllSources>/<EmbedUntrackedSources>true<\/EmbedUntrackedSources>/" $f
```

Comment:
Convert `EmbedAllSources` to `EmbedUntrackedSources` to keep same amount of exposure (no clue why this was initially desired)